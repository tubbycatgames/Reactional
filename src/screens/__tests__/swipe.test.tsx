import React from 'react'
import renderer from 'react-test-renderer'

import Swipe from '../swipe'

describe('Swipe', () => {
  it('should render', () => {
    const navigation = { goBack: () => {} }
    const render = renderer.create(<Swipe navigation={navigation} />)
    expect(render.toJSON()).toMatchSnapshot()
  })
})
