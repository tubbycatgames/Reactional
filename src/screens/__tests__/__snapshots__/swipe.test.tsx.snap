// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Swipe should render 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#e1f3f8",
      },
      Object {
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <View
    accessibilityLabel="Click to return to the Menu"
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Menu
      </Text>
    </View>
  </View>
</View>
`;
